legacy/messaging/src/main/java/org/jboss/as/messaging/MessagingTransformers.java

```diff
@@ -116,10 +87,21 @@ static void registerTransformers(final SubsystemRegistration subsystem) {
      */
     private static void buildTransformers2_1_0(ResourceTransformationDescriptionBuilder builder) {
         ResourceTransformationDescriptionBuilder hornetqServer = builder.addChildResource(pathElement(HORNETQ_SERVER));
-        rejectDefinedAttributeWithDefaultValue(hornetqServer, OVERRIDE_IN_VM_SECURITY);
-
         ResourceTransformationDescriptionBuilder addressSetting = hornetqServer.addChildResource(AddressSettingDefinition.PATH);
-        rejectDefinedAttributeWithDefaultValue(addressSetting, MAX_REDELIVERY_DELAY, REDELIVERY_MULTIPLIER, SLOW_CONSUMER_CHECK_PERIOD, SLOW_CONSUMER_POLICY, SLOW_CONSUMER_THRESHOLD);
+        rejectDefinedAttributeWithDefaultValue(addressSetting, MAX_REDELIVERY_DELAY, REDELIVERY_MULTIPLIER);
```

Comment:
This part was quite tricky to solve, but long story short.
we don't have consistent current --> previous --> previous --> ... chain here
but it is current --> previous (2.0) 
and than current --> last EAP6.4 one --> previous eap6 ...
result of stuff being added to 2.0(wildfly 8+) and eap 6.x at the same time.

so the stuff still gets properly transformed but just attributes are added do different part of chain where it is appropriate to have them.